internal/controller/pipelinerollout/pipelinerollout_controller_test.go

```diff
@@ -654,7 +654,7 @@ func Test_processExistingPipeline_PPND(t *testing.T) {
 			existingPipelineDef: *ctlrcommon.CreateTestPipelineOfSpec(
 				ctlrcommon.PipelineWithDesiredPhase(pipelineSpecWithTopologyChange, numaflowv1.PipelinePhasePaused),
 				ctlrcommon.DefaultTestPipelineName, numaflowv1.PipelinePhasePaused, numaflowv1.Status{},
-				false, map[string]string{}),
+				false, map[string]string{"numaplane.numaproj.io/isbsvc-name": "my-isbsvc", "numaplane.numaproj.io/parent-rollout-name": "pipelinerollout-test", "numaplane.numaproj.io/upgrade-state": "promoted"}),
```

Comment:
Hate to ask this, but I see we're adding these to `existingPipelineDef` here but not in the other test cases. For consistency, I wonder if we could. You know how most of them are using the function `createDefaultTestPipeline()`? What if we set these same labels in that function?